### DISCUSSION TITLE
ChaiNNer error installing

### DISCUSSION THREAD

**Discussion by xhyperzonex** [community/community#30356] [Other Feature Feedback, Questions, & Ideas] [answered]
hi i am getting an error like this when installing chainner. What do you think could be the reason?

![chrome_QYL06J48rP](https://user-images.githubusercontent.com/111731113/185861129-3724411e-c5e0-49ae-abe1-9ba9770113f6.png)

https://i.imgur.com/2ZTEwKe.png

**Comment by airtower-luna** [comment, marked as answer]
It looks like you're looking for help with a software hosted on GitHub, maybe https://github.com/joeyballentine/chaiNNer? In that case you should open an issue *in that repository* so the developers see it. They'll probably need a bit more details, though. :slightly_smiling_face:

**Reply by xhyperzonex** [reply]
Thank you so much
